src/test/java/com/datadog/api/v2/client/api/SecurityMonitoringApiTest.java

```diff
@@ -192,121 +194,181 @@ public void searchSignals() throws Exception {
             }
         });
 
-        // Sort works correctly
-        SecurityMonitoringSignalsListResponse responseAscending = api.searchSecurityMonitoringSignals()
-                .body(new SecurityMonitoringSignalListRequest()
-                        .filter(allSignalsFilter)
-                        .sort(SecurityMonitoringSignalsSort.TIMESTAMP_ASCENDING))
-                .execute();
+        // Sort ascending works correctly
+        TestUtils.retry(10, 10, () -> {
+            try {
+                SecurityMonitoringSignalsListResponse responseAscending = api.searchSecurityMonitoringSignals()
+                        .body(new SecurityMonitoringSignalListRequest()
+                                .filter(allSignalsFilter)
+                                .sort(SecurityMonitoringSignalsSort.TIMESTAMP_ASCENDING))
+                        .execute();
 
-        assertEquals(2, responseAscending.getData().size());
-        OffsetDateTime firstTimestamp = responseAscending.getData().get(0).getAttributes().getTimestamp();
-        OffsetDateTime secondTimestamp = responseAscending.getData().get(1).getAttributes().getTimestamp();
-        assertTrue(firstTimestamp.isBefore(secondTimestamp) || firstTimestamp.isEqual(secondTimestamp));
+                if (responseAscending.getData() == null || responseAscending.getData().size() < 2)
+                    return false;
```

Comment:
Can you add brackets here?

Reply:
yes good catch, I fixed all of them, my IDE save actions were disabled